packages/experiments/src/slots/examples/Slots.Icon.Example.tsx

```diff
@@ -11,20 +11,6 @@ export class SlotsIconExample extends React.Component<{}, {}> {
       <Stack {...stackProps}>
         <Button icon="share" content="Icon: String" />
         <Button icon={{ iconName: 'share' }} content="Icon: Props, iconName: 'share'" />
-        <Button
-          icon="upload"
-          slots={{
-            icon: {
-              render: (iconProps, DefaultComponent) => (
-                <b>
-                  Icon: <DefaultComponent {...iconProps} />
-                </b>
-              )
-            }
-          }}
-          content="Icon: Function, Text + Icon"
-        />
-        <Button slots={{ icon: { render: () => <Spinner /> } }} content="Icon: Function, Spinner" />
```

Comment:
I'm curious why these were removed. Did the changes in this PR reduce slot functionality?